cmd/agent/core/agent.go

```diff
@@ -88,7 +88,7 @@ func run(ctx context.Context, c *cli.Command, backends []types.Backend) error {
 		hostname, _ = os.Hostname()
 	}
 
-	counter.Polling = int(c.Int("max-workflows"))
+	counter.Polling = int(c.Int64("max-workflows"))
```

Comment:
Instead of casting could we use `Int(` directly?

Reply:
Good catch, yes now we can again.

Reply:
I changed a few more type casts to `IntFlag`.